inference-engine/tests_deprecated/fluid_preproc/fluid_test_computations/fluid_test_computations.hpp

```diff
@@ -104,4 +107,9 @@ class FLUID_COMPUTATION_VISIBILITY ConvertDepthComputation : public FluidComputa
     ConvertDepthComputation(test::Mat inMat, test::Mat outMat, int depth);
 };
 
+bool FLUID_COMPUTATION_VISIBILITY preprocessWithGAPI(
```

Comment:
need to add //FIXME remove than PP streamlining patch lands (https://github.com/openvinotoolkit/openvino/pull/857)